save-agent/src/jvmMain/kotlin/com/saveourtool/save/agent/utils/Logging.kt

```diff
@@ -0,0 +1,23 @@
+/**
+ * Logging utilities for save-agent
+ */
+
+@file:JvmName("LoggingJVM")
+
+package com.saveourtool.save.agent.utils
+
+import com.saveourtool.save.core.logging.logDebug
+import com.saveourtool.save.core.logging.logError
+import com.saveourtool.save.core.logging.logInfo
+
+actual fun logErrorCustom(msg: String) = logError(
+    "[tid ${Thread.currentThread().id}] $msg"
```

Comment:
maybe make method getTid with different implementation instead of the full method log?

Reply:
Good idea. fixed